recipe/meta.yaml

```diff
@@ -1,52 +1,125 @@
-{% set name = "ariadne-codegen" %}
-{% set version = "0.13.0" %}
+{% set version = "0.14.0" %}
+
+# handle undefined PYTHON in `noarch: generic` outputs
+{% if PYTHON is not defined %}{% set PYTHON = "$PYTHON" %}{% endif %}
```

Comment:
I don't really understand why this is needed

Reply:
Ah I think I got it. The build has to be moved from package-level to the first output for the `PYTHON` variable to be set.

Reply:
... and there needs a global Python host and run requirement, I took that from https://github.com/conda-forge/gql-feedstock/blob/main/recipe/meta.yaml

Reply:
... as well as in every subpackage

Reply:
the `noarch: generic` saves the extra solves for each package, which is a better use of CI resources.

Reply:
xref https://github.com/conda/conda-build/issues/5424

Reply:
Agreed, see #3